test/typebox-nested-schema.test.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const test = require('tap').test
+const build = require('..')
+
+const { Type } = require('@sinclair/typebox')
+
+test('nested object in pattern properties for typebox', (t) => {
```

Comment:
Let skip it on `node@10`
```suggestion
test('nested object in pattern properties for typebox', { skip: semver.lt(process.versions.node, '12.0.0') }, (t) => {
```